configure.ac

```
@@ -6873,3 +6938,9 @@ If you want a release build with all stable optimizations active (PGO, etc),
 please run ./configure --enable-optimizations
 ])
 fi
+
+AS_VAR_IF([PY_SUPPORT_TIER], [0], [AC_MSG_NOTICE([
```

Ditto; I'd use a warning for added emphasis.

```suggestion
AS_VAR_IF([PY_SUPPORT_TIER], [0], [AC_MSG_WARN([
```